containers/runner/scenario

```diff
@@ -2,14 +2,26 @@
 # Launch a scenario for the daily run.
 set -eux
 
-LAUNCH=$(dirname "$0")/launch
+MYDIR=$(dirname $(realpath "$0"))
+LAUNCH="$MYDIR/launch"
+ROOTDIR=$(dirname $(dirname "$MYDIR"))
 SCENARIO="$1"
 shift
 
 case "$SCENARIO" in
     rawhide) $LAUNCH --skip-testtypes rhel-only,knownfailure "$@" all ;;
     rawhide-text) $LAUNCH --skip-testtypes rhel-only,knownfailure --run-args '-eKSTEST_EXTRA_BOOTOPTS=inst.text' "$@" all ;;
     daily-iso) $LAUNCH --skip-testtypes rhel-only,knownfailure --testtype packaging --daily-iso="$GITHUB_TOKEN" "$@" ;;
+
+    rhel8)
+        if [ ! -e data/images/boot.iso ]; then
+            echo "INFO: data/images/boot.iso does not exist, downloading current RHEL 8 boot iso..."
+            mkdir -p data/images
+            curl -L http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8/compose/BaseOS/x86_64/os/images/boot.iso --output data/images/boot.iso
+        fi
+        $LAUNCH --skip-testtypes rhel-only,knownfailure --testtype packaging --defaults "$ROOTDIR/scripts/defaults-rhel8.sh"
```

Comment:
I think we want "fedora-only" instead of "rhel-only" here.

Reply:
Argh, of course! Fixed. Can you please re-approve?